### DISCUSSION TITLE
如何较为简单添加一个流程判断的跳转条件

### DISCUSSION THREAD

**Discussion by cfc3058353263** [didi/LogicFlow#1313] [General]
![image](https://github.com/didi/LogicFlow/assets/52273637/189adc08-326e-4b49-9785-b3f85ebf1f37)
想要在流程线中通过如此方法添加一个流程的判断条件请问如何添加，有示例吗

**Comment by wumail** [comment]
这个判断条件要干嘛啊，只需要最终体现在生成的xml里？

**Reply by cfc3058353263** [reply]
我是想要在bpmn:sequenceFlow中添加条件判断，最终在xml中显示的，不知道使用什么方法，看到官网的实例中adapterOut 的 方法不太懂，不知道有没有些简单的实例参考一下

**Reply by wumail** [reply]
可以看下这个demo https://docs.logic-flow.cn/demo/dist/bpmn-vue/

**Comment by cfc3058353263** [comment]
![1694143463420](https://github.com/didi/LogicFlow/assets/52273637/061207ae-6a69-4448-a340-8d5b20664d50)
请问如何修改xsi:type中数据，希望把bpmn2 改为bpmn

**Reply by wumail** [reply]
修改一下transformer
```typescript
LogicFlow.use(BPMNAdapter, {
  transformer: {
    "bpmn:sequenceFlow": {
      out(data: any) {
        const {
          properties: { condition },
        } = data;
        if (condition) {
          return {
            json: `<bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${condition}</bpmn:conditionExpression>`,
          };
        }
        return {
          json: "",
        };
      },
    },
  },
});
```

**Comment by wumail** [comment]
https://juejin.cn/post/7275550548944764965 有位朋友发了一篇新BPMN插件相关的文章，感觉可以作为教程用了😂
